apps/docs/content/icons/getting-started/installation.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Installation
+description: Getting started with Workleap Design Icons
+order: 1
+---
+
+export const installMethods = [
+  { title: "pnpm" },
+  { title: "yarn" },
+  { title: "npm" }
+];
+
+# Installation
+
+Using your package manager of choice, run the following command to install the icons package.
```

Comment:
I wouldn't talk about package manager choices. I think this should be a paragraph that explains that Workleap icons are available as React components in the icons packages, and as plain svg images in the svg-icons package. with links to the appropriate section.